frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboards.ts

```diff
@@ -181,80 +231,92 @@ export const isDashboardsOverviewHealthPrometheusSubsystem = (
   e: Extension,
-): e is DashboardsOverviewInventoryItem => e.type === 'console.dashboards/inventory/item';
+): e is DashboardsOverviewInventoryItem => e.type === 'console.dashboards/overview/inventory/item';
+
+export const isDashboardsOverviewInventoryItemReplacement = (
+  e: Extension,
+): e is DashboardsOverviewInventoryItemReplacement =>
+  e.type === 'console.dashboards/overview/inventory/item/replacement';
+
+export const isProjectDashboardInventoryItem = (e: Extension): e is ProjectDashboardInventoryItem =>
+  e.type === 'console.dashboards/project/overview/item';
 
 export const isDashboardsOverviewResourceActivity = (
   e: Extension,
 ): e is DashboardsOverviewResourceActivity =>
   e.type === 'console.dashboards/overview/activity/resource';
 
-export type DashboardCardSpan = 4 | 6 | 12;
-
-export type GetOperatorsWithStatuses<R extends K8sResourceCommon = K8sResourceCommon> = (
-  resources: FirehoseResourcesResult,
-) => OperatorStatusWithResources<R>[];
-
-export type K8sActivityProps<R extends K8sResourceCommon = K8sResourceCommon> = {
-  resource: R;
-};
-
-export type OperatorRowProps<R extends K8sResourceCommon = K8sResourceCommon> = {
-  operatorStatus: OperatorStatusWithResources<R>;
-};
-
-export type OperatorStatusWithResources<R extends K8sResourceCommon = K8sResourceCommon> = {
-  operators: R[];
-  status: OperatorStatusPriority;
-};
-
-export type OperatorStatusPriority = {
-  title: string;
-  priority: number;
-  icon: React.ReactNode;
-  health: keyof typeof HealthState;
-};
-
-export type PrometheusHealthHandler = (
-  responses: { response: PrometheusResponse; error: any }[],
-  t?: TFunction,
-  additionalResource?: FirehoseResult<K8sResourceCommon | K8sResourceCommon[]>,
-) => SubsystemHealth;
+export const isDashboardsOverviewPrometheusActivity = (
+  e: Extension,
+): e is DashboardsOverviewPrometheusActivity =>
+  e.type === 'console.dashboards/overview/prometheus/activity/resource';
 
-export type PrometheusHealthPopupProps = {
-  responses: { response: PrometheusResponse; error: any }[];
-  k8sResult?: FirehoseResult<K8sResourceCommon | K8sResourceCommon[]>;
-};
+export type DashboardsOverviewHealthSubsystem =
+  | DashboardsOverviewHealthURLSubsystem
+  | DashboardsOverviewHealthPrometheusSubsystem
+  | DashboardsOverviewHealthResourceSubsystem
+  | DashboardsOverviewHealthOperator;
 
-export type ResourceHealthHandler<R extends ResourcesObject> = (
-  resourcesResult: WatchK8sResults<R>,
-  t?: TFunction,
-) => SubsystemHealth;
+export const isDashboardsOverviewHealthSubsystem = (
+  e: Extension,
+): e is DashboardsOverviewHealthSubsystem =>
+  isDashboardsOverviewHealthURLSubsystem(e) ||
+  isDashboardsOverviewHealthPrometheusSubsystem(e) ||
+  isDashboardsOverviewHealthResourceSubsystem(e) ||
+  isDashboardsOverviewHealthOperator(e);
 
-export type SubsystemHealth = {
-  message?: string;
-  state: HealthState;
+type DashboardsOverviewInventoryItemProperties<
+  T extends K8sKind = K8sKind,
```

Comment:
Can we replace all `K8sKind` for `GroupVersionKind` type ? I think it is used only for identifing k8s resource, where group-version-kind is enough.